infrastructure/api-configuration/nginx_config.conf

```diff
@@ -0,0 +1,89 @@
+# Set this to grep processor /proc/cpuinfo | wc -l
+worker_processes  2;
```

Comment:
So you ran `grep processor /proc/cpuinfo | wc -l`, it returned `2` and you set this here? Is there anyway to automate this? I could see us deciding at some point we need to beef up our API server, forgetting to change this, and then being confused as to why we aren't seeing the performance increase we expected.